src/app/features/section/shared/list/member-list/member-list.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-list class="list">
+  <ng-container *ngIf="sectionDetails$ | async as sectionDetails">
+    <app-members-list-item
+      *ngFor="let user of sectionDetails.users"
+      [name]="getFullName(user)"
+      [initials]="getInitials(user)"></app-members-list-item>
```

Comment:
a może po prostu przekażmy tam user'a i komponent już ogarnie na jego podstawie inicjaly i wyciągnie sobie imie? z tego co widzę jest to komponent domenowy więć możemy przekazać domenowego propsa dla niego